deploy.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# This script is intended to be run inside a docker container to provide a
+# hermetic process. See release.sh for the expected invocation.
+
+set -e
+
+# By default we deploy to both ossrh and bintray. These two
+# environment variables can be set to skip either repository.
+DEPLOY_OSSRH="${DEPLOY_OSSRH:-true}"
+DEPLOY_BINTRAY="${DEPLOY_BINTRAY:-false}"
```

Comment:
The comment says it deploys to both bintray and ossrh, but doesn't this line turn off bintray by default?

Reply:
I'm thinking now about removing completely support for Bintray. The deployment process I'm putting in place uses exclusively OSSRH so keeping this support for Bintray just bring noise in the scripts.

Reply:
Maybe commit it with bintray then we can remove it later. That way it's in the repo history and we can get it if necessary.

Reply:
It is already there, I'll take care of not squashing the commits on merge and it will remain in history.